agents/ppo.py

```diff
@@ -42,48 +48,57 @@ def create_from_config(cls, config):
             critic_loss_weight=config.get('critic_loss_weight', 1.0)
         )
 
-    def forward(self, state: State) -> Tuple[State, Policy, Value, Value]:
-        next_state, policy = self._actor.forward(state)
-        curr_v_func = self._critic.forward(state)  # (batch_size)
-        q_estimate = self._critic.forward(next_state)  # (batch_size)
-        # TODO [Vladimir Baikalov]: make same inerface for every agent's forward function
-        return next_state, policy, curr_v_func, q_estimate
+    def forward(self, inputs: Dict[str, Any]):
```

Comment:
Let's add the output type as well
```suggestion
    def forward(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
```